meteor/lib/collections/Parts.ts

```diff
@@ -100,6 +100,7 @@ export class Part implements DBPart {
         color?: string
     }
 	public floated?: boolean
+	public gap?: boolean
```

Comment:
Just a note; this PR is dependent on an addition of the `IBlueprintPart.gap`-property to the blueprint-integration repo as well

Reply:
Yes, and now the blueprint-integration has been updated.